interpreter.py

```diff
@@ -1,4 +1,4 @@
-import sys; O="«»><+-,.[]()      ≥≤  ;:"; Oi=O.index
+import sys; O="«»><+-,.[]()±∓    ≥≤  ;:"; Oi=O.index
```

Comment:
I was thinking of pairing `±` with `+` and `∓` with `-`, so maybe move them to positions `20` and `21`? This way `±` and `∓` can be seen as modified `+` and `-` respectively.